THEME.md

```diff
@@ -205,25 +205,46 @@ It's a good idea to add this to the top of the template to improve the styling o
 
 ## Action Injection
 
-The following will inject the action link (or button) into the e-mail:
+The following will inject the action link (or button) into the e-mail.
 
 ```html
 <!-- Action -->
+<% if(locals.action.constructor !== Array){ %>
```

Comment:
Also, the conversion of the `locals.action` to an array should be done at the top of the theme files (I see that you have done that but have included it here as well).